config/v1/types_authentication.go

```diff
@@ -243,11 +244,29 @@ type OIDCProvider struct {
 	// +listType=atomic
 	// +optional
 	ClaimValidationRules []TokenClaimValidationRule `json:"claimValidationRules,omitempty"`
+
+	// userValidationRules defines the set of rules used to validate claims in a user's token.
+	// Each rule is evaluated independently to determine whether the token subject is considered valid.
+	// Rules can either require specific claims and values to be present,
+	// or define CEL expressions that must evaluate to true for the token to be accepted.
+	// All rules must evaluate to true for the token to be accepted.
+	// If any rule evaluates to false, the token is rejected.
+	// A maximum of 64 rules can be specified. This field is optional.
+	//
+	// See https://kubernetes.io/docs/reference/using-api/cel/ for CEL syntax.
```

Comment:
Don't include links in the documentation of a field where possible. Links get stale and inclusion in the generated documentation makes it difficult for users to know what was intended to be shown if the link is no longer valid or has moved.